support-config/src/main/scala/com/gu/support/config/SafeLogger.scala

```diff
@@ -0,0 +1,46 @@
+package com.gu.support.config
+
+import com.typesafe.scalalogging.StrictLogging
+import org.slf4j.{Marker, MarkerFactory}
+
+object SafeLogger extends StrictLogging {
```

Comment:
This class already exists in `support-services`. Since you are adding it here and support-services depends on this library I think you can remove it from there.

Reply:
good idea

Reply:
I have also moved the new one to the equivalent old package to avoid messing up the old code's important - that's pushed now